packages/playground/src/components/k8s_deployment_table.vue

```diff
@@ -85,9 +84,15 @@
       </template>
       <template #[`item.status`]="{ item }">
         <v-chip :color="getNodeHealthColor(item.value.masters[0].status as string).color">
-          <v-tooltip v-if="item.value.masters[0].status == NodeHealth.Error" activator="parent" location="top">{{
-            item.value.masters[0].message
-          }}</v-tooltip>
+          <v-tooltip
+            v-if="item.value.masters[0].status == NodeHealth.Error || item.value.masters[0].status == NodeHealth.Paused"
```

Comment:
in case NodeHealth.Error the tooltip message shouldn't be ``` The deployment contract is in grace period```

#### Suggestions:
- Tooltip message should reflect the error
- Remove the tooltip in case health=error